### ISSUE TITLE
Nesse repositório só tem os PDF's até o capítulo 20

### ISSUE THREAD

**Issue by thaisdss** [gustavoguanabara/html-css#243]

**Comment by thaisdss** [comment]
Boa tarde, nesse repositório de aulas tem apenas os PDF's das aulas até o capítulo 20

**Comment by filipegomespires** [comment]
Boa tarde, não existe mais material em PDF além do capítulo 20 ?
Obrigado

**Comment by flameastro** [comment]
@thaisdss Sim, vai até o capítulo 20... Por favor, certifique-se de fechar essa issue!

Agradeço!
